bld/namelist_files/use_cases/2010_control.xml

```diff
@@ -8,44 +8,17 @@
 
 <sim_year_range>constant</sim_year_range>
 
-<irrigate use_crop=".true." phys="clm5_1"  use_cndv=".false.">.true.</irrigate>
-<irrigate use_crop=".true." phys="clm5_1"  use_cndv=".false.">.true.</irrigate>
-<irrigate use_crop=".true." phys="clm5_1"  use_cndv=".true." >.false.</irrigate>
-<irrigate use_crop=".true." phys="clm5_0"  use_cndv=".false.">.true.</irrigate>
-<irrigate use_crop=".true." phys="clm5_0"  use_cndv=".true." >.false.</irrigate>
-<irrigate use_crop=".true." phys="clm4_5"                    >.false.</irrigate>
+<irrigate use_crop=".true." =".false."    >.true.</irrigate>
+<irrigate use_crop=".true." =".false."    >.true.</irrigate>
+<irrigate use_crop=".true." phys="clm4_5" >.false.</irrigate>
```

Comment:
This setting ends up being different from ctsm5.1.dev176 for clm4_5, but I think the new version is correct. So leaving this as it is.